src/ott/solvers/linear/sinkhorn.py

```diff
@@ -409,7 +410,8 @@ def converged(self) -> bool:  # noqa: D102
     if self.errors is None:
       return False
     return jnp.logical_and(
-        jnp.any(self.errors == -1), jnp.all(jnp.isfinite(self.errors))
+        jnp.logical_not(jnp.any(jnp.isnan(self.errors))),
```

Comment:
Since we're refactoring this - wouldn't it be better to have a `converged` variable directly in the `SinkhornOutput` rather than recomputing it always here?

Reply:
If not, then please just add the comment I requested below and merge!

Reply:
sure, used your suggestion to pre-compute rather than use as a property